apps/server/buffer.proto

```diff
@@ -58,6 +58,7 @@ message ClientRequest
   repeated bytes func_bytes_param = 40; // List of function parameters of type byte-array
   bool expect_reply = 50;               // Whether a response with data is expected or not
   FrameDetails frame_type = 60;         // Frame type information
+  int32 id = 70;
```

Comment:
It would make more sense to follow the structure of the other sensors (temperature, storage).
I mean have another message called ImagerInfo with fields name and id. And add it to the SensorsInfo message.